src/index.ts

```diff
@@ -130,15 +130,22 @@ function VitePluginWindicss(options: Options = {}): Plugin[] {
         classesPending.add(i)
       })
 
+    function addTag(i: string) {
+      if (i === 'router-link')
+        i = 'a'
```

Comment:
Prefer using a map, with the ability to be extended by user-config.

Reply:
Also it's also possible to have something like `RouterLink`, will need to apply a normalization, I'd recommended do it in the `generateCSS` function so it has less overhead.